src/Http/Routing/src/EndpointMiddleware.cs

```diff
@@ -101,28 +101,13 @@ private static void ThrowMissingCorsMiddlewareException(Endpoint endpoint)
                 "Configure your application startup by adding app.UseCors() inside the call to Configure(..) in the application startup code. The call to app.UseCors() must appear between app.UseRouting() and app.UseEndpoints(...).");
         }
 
-#nullable disable
-        private static class Log
+        private static partial class Log
         {
-            private static readonly Action<ILogger, string, Exception> _executingEndpoint = LoggerMessage.Define<string>(
-                LogLevel.Information,
-                new EventId(0, "ExecutingEndpoint"),
-                "Executing endpoint '{EndpointName}'");
+            [LoggerMessage(0, LogLevel.Information, "Executing endpoint '{EndpointName}'", EventName = "ExecutingEndpoint")]
```

Comment:
Is 0 really an Id we want to use?

Reply:
That would be a breaking change. I can understand making one because it was problematic, but breaking it intentionally seems unnecessary.